server/src/utils/DiscordUtils.ts

```diff
@@ -215,6 +220,24 @@ ${error.stack}
     });
   }
 
+  public sendAscendNotification(brute: Pick<Brute, 'name'>, ascensions: number) {
+    const embed = new EmbedBuilder()
+      .setColor(0xeb8770)
+      .setTitle(formatEmbedTitle(`${brute.name} ascended ${translate(`rank.${ascensions}`)} times`))
+      .setURL(`${this.#server}${brute.name}/cell`)
+      .setAuthor({
+        name: 'LaBrute',
+        iconURL: `${this.#server}/favicon.png`,
+        url: `${this.#server}${brute.name}/ranking`,
+      })
+      .setThumbnail(`${this.#server}/images/rankings/lvl_0.webp`)
+      .setTimestamp();
+
+    this.#rankUpClient.send({ embeds: [embed] }).catch((err) => {
```

Comment:
@Zenoo may need to change this if you wish, not sure i can really do much about discord notifications

Reply:
What do you mean? Everything looks good to me here